server/controllers/auth.js

```diff
@@ -13,17 +13,17 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
 
   if (emailExists) {
     res.status(400);
-    throw new Error("A user with that email already exists");
+    throw new Error('A user with that email already exists');
   }
 
   const usernameExists = await User.findOne({ username });
 
   if (usernameExists) {
     res.status(400);
-    throw new Error("A user with that username already exists");
+    throw new Error('A user with that username already exists');
   }
-  const newProfile = await Profile.create();
-  const profile = newProfile._id
+  const newProfile = await Profile.create({});
```

Comment:
Just so we are on the same page, this call **will** fail if there are required fields in the `Profile` model without default values. Currently, there aren't any so nothing to worry about (and FWIW, I like this implementation better 👍🏼 ) but we should be mindful of this if/when updating the model.

Reply:
@ugur-akin Noted!